ocp_resources/data_import_cron.py

```diff
@@ -25,22 +21,26 @@ def __init__(
         managed_data_source=None,
         imports_to_keep=None,
         bind_immediate_annotation=None,
-        teardown=True,
-        privileged_client=None,
-        yaml_file=None,
-        delete_timeout=TIMEOUT_4MINUTES,
         **kwargs,
     ):
-        super().__init__(
-            name=name,
-            namespace=namespace,
-            client=client,
-            teardown=teardown,
-            privileged_client=privileged_client,
-            yaml_file=yaml_file,
-            delete_timeout=delete_timeout,
-            **kwargs,
-        )
+        """
+        Args:
+            garbage_collect (str, optional): whether old PVCs should be cleaned up after a new PVC is imported.
+                Options are "Outdated"/"Never". Default value is "Outdated".
```

Comment:
```suggestion
                Options are "Outdated"/"Never". 
```
This is not set by the code here